internal/cmdsearch/searchreplace_test.go

```diff
@@ -624,7 +624,7 @@ spec:
 	},
 	{
 		name: "put literal error",
-		args: []string{"--put-literal", "something"},
+		args: []string{"--put-value", "something"},
```

Comment:
is there a test for put-value containing capture groups?

Reply:
Added. Had a test with both put-value and put-comment but added another test with just put-value containing capture groups.